.github/workflows/codespell.yml

```diff
@@ -18,4 +18,6 @@ jobs:
       - uses: codespell-project/actions-codespell@master
         with:
           exclude_file: .codespell-exclude-lines
-          skip: ./repos/system_upgrade/common/actors/storagescanner/tests/files/mounts
+          ignore_words_list: ro,fo
+          skip: "./repos/system_upgrade/common/actors/storagescanner/tests/files/mounts,\
```

Comment:
Do we really need to skip some files? Or is it a temporary measure until you catch all issues and fixup into the proper comments?

Reply:
Yeah we do, for example `./repos/system_upgrade/common/actors/storagescanner/tests/files/mounts` contains just mount points and mount options so there aren't really any words which should be checked. They would just create unnecessary false positives.

Reply:
Thanks or clarification!